docs/integrations/microsoft-azure/opentelemetry/active-directory-json-opentelemetry.md

```diff
@@ -0,0 +1,140 @@
+
+We recommend using the Active Directory JSON App in combination with the Windows JSON App.Active Directory logs are sent to Sumo Logic through OpenTelemetry [windowseventlogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowseventlogreceiver).
+
+<img src={useBaseUrl('https://https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Active-Directory-OpenTelemetry/Active-Directory-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Active Directory
+
+Following are the [fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Active Directory App install if not already present.
+
+**`sumo.datasource`** - Has fixed value of **activeDirectory**
+
+### Event Logs Used by Active Directory App
+
+This section provides instructions for configuring log collection for Active Directory running on a non-Kubernetes environment for the Sumo Logic App for Active Directory.
+
+#### Log Types
+
+Standard Windows event channels include:
+
+-   **Security**
+-   **System**
+-   **Application**
+
+## Collection Configuration & App installation
+
+As part of the setting up the collection process and app installation, user can select the App from App Catalog and click on Install App. Please follow the steps below:
+
+### Step 1: Set up Collector
+
+If you want to use an existing OpenTelemetry Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+This will generate a command which can be executed in the windows machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Active-Directory-OpenTelemetry/Active-Directory-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for Active Directory Collection.
+
+You can add any additional fields which you want to tag as part of the data collected.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Active-Directory-OpenTelemetry/Active-Directory-YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps
+
+1.  Copy the yaml at `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the machine which needs to be monitored.
+2.  restart the collector using the command `Restart-Service -Name`.
```

Comment:
```suggestion
2.  Restart the collector using the command `Restart-Service -Name`.
```

Reply:
Not sure about the hyphen (-Name). I don't think there should be a space. @nmohantyctr , can you confirm this?